include/libtorrent/aux_/openssl.hpp

```diff
@@ -101,6 +105,6 @@ inline GENERAL_NAME* openssl_general_name_value(GENERAL_NAMES* gens, int i)
 
 #include "libtorrent/aux_/disable_warnings_pop.hpp"
 
-#endif // TORRENT_USE_OPENSSL
+#endif // TORRENT_SSL_PEERS
```

Comment:
This comment doesn't look right. I would expect the corresponding `#if` to also have changed in this file

Reply:
you are right